compiler/rustc_target/src/spec/riscv64gc_unknown_hermit.rs

```diff
@@ -0,0 +1,21 @@
+use crate::spec::Target;
+use crate::spec::{CodeModel, RelocModel, TlsModel};
+
+pub fn target() -> Target {
+    let mut base = super::hermit_base::opts();
+    base.cpu = "generic-rv64".to_string();
+    base.max_atomic_width = Some(64);
+    base.features = "+m,+a,+f,+d,+c".to_string();
+    base.code_model = Some(CodeModel::Medium);
+    base.relocation_model = RelocModel::Pic;
+    base.tls_model = TlsModel::LocalExec;
```

Comment:
Why is this TLS model used by default? Does the target not support `InitialExec`?

Reply:
I don't think that `InitialExec` is needed since the target is not used to build shared libraries.